test_tortoise_interpreter.py

```diff
@@ -4,45 +4,77 @@
     main_tortoise,
     sanitize_program,
     navigate_command_functions,
-    pen_up, pen_colour,
-    )
+    pen_up,
+    pen_colour,
+)
 
 
 def test_main_tortoise():
     program = [
-        'P 2  # select pen 2',
-        'D    # pen down',
-        'W 2  # draw west 2cm',
-        'N 1  # then north 1',
-        'E 2  # then east 2',
-        'S 1  # then back south',
-        'U    # pen up'
+        "P 2  # select pen 2",
+        "D    # pen down",
+        "W 2  # draw west 2cm",
+        "N 1  # then north 1",
+        "E 2  # then east 2",
+        "S 1  # then back south",
+        "U    # pen up",
     ]
     program = [list(i.strip()) for i in program]
 
     list_of_strings = []
+
     def print_function(stuff_to_print):
         list_of_strings.append(stuff_to_print)
 
     main_tortoise(program, print_function)
 
-    assert list_of_strings == ['1. P 2', '2. PEN DOWN', '3. W 2', '4. N 1', '5. E 2', '6. S 1', '7. PEN UP']
-
-
-@pytest.mark.parametrize('program,                         expected', [
-    (['P 3', 'U', 'D'],                                    [['P', '3'], ['U'], ['D']]),
-    (['P 3', 'U', 'D', 'P 3', 'U', 'D'],                   [['P', '3'], ['U'], ['D'], ['P', '3'], ['U'], ['D']]),
-    ([ 'P 3', 'U', 'D', 'P 3', 'U', 'D', 'P 3', 'U', 'D'], [['P', '3'], ['U'], ['D'], ['P', '3'], ['U'], ['D'], ['P', '3'], ['U'], ['D']]),
-    ([
-        'P 2  # select pen 2',
-        'D    # pen down',
-        'W 2  # draw west 2cm',
-        'N 1  # then north 1',
-        'E 2  # then east 2',
-        'S 1  # then back south',
-        'U    # pen up'
-    ],                                                     [['P', '2'], ['D'], ['W', '2'], ['N', '1'], ['E', '2'], ['S', '1'], ['U']]),
-])
+    assert list_of_strings == [
+        "1. P 2",
+        "2. PEN DOWN",
+        "3. W 2",
+        "4. N 1",
+        "5. E 2",
+        "6. S 1",
+        "7. PEN UP",
+    ]
+
+
+@pytest.mark.parametrize(
+    "program,                         expected",
+    [
+        (["P 3", "U", "D"], [["P", "3"], ["U"], ["D"]]),
+        (
+            ["P 3", "U", "D", "P 3", "U", "D"],
+            [["P", "3"], ["U"], ["D"], ["P", "3"], ["U"], ["D"]],
+        ),
+        (
+            ["P 3", "U", "D", "P 3", "U", "D", "P 3", "U", "D"],
+            [
+                ["P", "3"],
+                ["U"],
+                ["D"],
+                ["P", "3"],
+                ["U"],
+                ["D"],
+                ["P", "3"],
+                ["U"],
+                ["D"],
+            ],
+        ),
+        (
+            [
+                "P 2  # select pen 2",
+                "D    # pen down",
+                "W 2  # draw west 2cm",
+                "N 1  # then north 1",
+                "E 2  # then east 2",
+                "S 1  # then back south",
+                "U    # pen up",
+            ],
+            [["P", "2"], ["D"], ["W", "2"], ["N", "1"], ["E", "2"], ["S", "1"], ["U"]],
+        ),
+    ],
+)
```

Comment:
If you think this formatting was better before, you can disable it with (I think) `# noqa`

Reply:
`# fmt: off` and `# fmt: on` is the pylint version it seems.